config/initializers/constants.rb

```diff
@@ -0,0 +1 @@
+USE_TRANSFER_INITIATIVE_MESSAGE = "Use Initiative#transfer_to instead".freeze
```

Comment:
You should never use a file in initializers to create a global constant because you should not have a global constant for a string